CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## Unreleased
+
+### Fixes
+
+- Fix thread inversion warning in session replay (#5018)
```

Comment:
`m`: Yes this fixes the warning, but what's the customer benefit? Faster video conversion and no deadlocks or such? If yes, can we rephrase this?

Reply:
Updated, should be good now